lisa/main.py

```diff
@@ -17,7 +17,7 @@
 from lisa.variable import add_secrets_from_pairs
 
 
-@retry(FileExistsError, tries=10, delay=0.2)
+@retry(FileExistsError, tries=10, delay=0.2)  # type: ignore
```

Comment:
Weird, it *introduces* mypy errors?? What about the other type-whatever packages? Don't they help remove any of these?

Reply:
Not sure what's difference between retry and other types packages, but other packages works well. This fix makes the mypy passed.